libraries/core-react/src/components/TopBar/TopBar.tokens.ts

```diff
@@ -27,4 +28,9 @@ export const token: TopBarToken = {
       width: '2px',
     },
   },
+  modes: {
+    compact: {
+      height: shape._modes.compact.straight.minHeight,
```

Comment:
I would remove this as the height is an exception for the default value for this token for `Topbar`, since it so much taller then our other components (64px). We will probably have to do abit more in terms of compact here as just adjusting the height, looks very wrong 🙈 

![image](https://user-images.githubusercontent.com/1070981/144840734-97874929-627a-4082-a3ee-866a005d500f.png)